src/control/cmd/dmg/pool.go

```diff
@@ -204,6 +236,104 @@ func (cmd *PoolCreateCmd) Execute(args []string) error {
 	return nil
 }
 
+// XXX DAOS-9196 Some extra bytes are needed for the metadata of the pool: at least 135 MB
+// are needed.  The extraByes is a little bigger than this limit, because the size of the
+// metadata will eventually grow along the life of the pool.
+const PoolMetadataBytes uint64 = uint64(200) * uint64(humanize.MiByte)
```

Comment:
Per Go conventions, comments on exported symbols should begin with the name of the entity, e.g. `// PoolMetadataBytes defines the amount of storage reserved for pool metadata.`. This allows for automatically-generated documentation. If the symbol is not relevant outside of the package, then it should not be exported (i.e. the first letter should be lowercase).

Reply:
Following tasks will be done:
- [ ] Fix code documentation to respect go idiomatic coding convention